### ISSUE TITLE
Multiple correct answers

### ISSUE THREAD

**Issue by chandru1782** [gbateson/moodle-qtype_ordering#78]
Hi, 

Is it possible to add the option of supporting multiple correct answers in this type of question? For example, I would like to award grade point when it is either `Item m` followed by `Item n` or vice versa. That is the orderings "`Item m` followed by `Item n`" and "`Item n` followed by `Item m`" are both correct. 

Please help and let me know if I can be of any use.

Thanks,
Chandru

**Comment by gbateson** [comment]
I see. Perhaps one way of doing this would be to allow multiple versions of the "correct" sequence. Then, when grading a student's response, we can work out the grades compared to each "correct" sequence and award the highest of those grades.

Could you give a concrete example? I'd like some idea of how many items are required? And how many "correct" sequences would suit your needs?

**Comment by chandru1782** [comment]
Hi, Thank you for the reply. We try to use these type of questions for students of Mathematics. Assume that a student has to put the following sets in the order of inclusion.

1. Empty set.
2. The set of all real numbers whose square is negative.
3. The set of all real numbers whose square is zero.

Here both sets in items 1 and 2 are empty. Therefore the orderings 1,2,3 and 2,1,3 are both correct. 

In summary I would like to say that we might have to add provision for at most three different correct orderings.
